src/client/pages/Offer/InsuranceSelector/index.tsx

```diff
@@ -59,46 +65,68 @@ export const InsuranceSelector = ({
     [variants],
   )
 
+  const exposure = useMemo(() => getInsuranceExposure(variants), [variants])
```

Comment:
Do we need separate memo for exposure? It seems to be used once, maybe we can inline it

Reply:
It's used in two places -- does that make a difference? 😅